testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/client/ClientResponseRedirectTest.java

```diff
@@ -46,10 +46,12 @@
         WebArchive war = TestUtil.prepareArchive(ClientResponseRedirectTest.class.getSimpleName());
         war.addClass(ClientTestBase.class);
         // Use of PortProviderutil in the deployment
-        war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(new PropertyPermission("node", "read"),
+        war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(
+                new PropertyPermission("node", "read"),
                 new PropertyPermission("ipv6", "read"),
                 new RuntimePermission("getenv.RESTEASY_PORT"),
-                new PropertyPermission("org.jboss.resteasy.port", "read")), "permissions.xml");
+                new PropertyPermission("org.jboss.resteasy.port", "read")
```

Comment:
Actually, now that I notice this, I wondering if running the System.getProperty("org.jboss.resteasy.port") in a privileged block wouldn't be better than asking users to explicitly add the permissions...

Reply:
The test calls PortProviderUtil in arquillian-utils.  This class is calling 
System.getenv and/or System.getProperty however putting these calls in a privileged 
block is of no help because the test itself is not running in a security manager.

Reply:
Hmmm. If the client is not running in the SM why is the permission being requested.    This is curious.

Reply:
It looks like wildfly security implements a rule that System.getProperty and
System.getenv calls require explicit permissions to exec the cmd.
15:27:43,036 ERROR [stderr] (default task-1) 	at org.wildfly.security.manager.WildFlySecurityManager.checkPermission(WildFlySecurityManager.java:295)
15:27:43,036 ERROR [stderr] (default task-1) 	at org.wildfly.security.manager.WildFlySecurityManager.checkPermission(WildFlySecurityManager.java:192)

This means the existing permission 
          new PropertyPermission("node", "read"),
          new PropertyPermission("ipv6", "read"),
          new RuntimePermission("getenv.RESTEASY_PORT"),
          new PropertyPermission("org.jboss.resteasy.port", "read")

must stay unless there is some other way to direct wildfly to allow them.

Reply:
Interesting, can you please investigate this a little bit more? Does it have anything to do with arquillian being used to run the tests?

Reply:
OK, right. What's not clear to me is why the permission check fails if you wrap the invocation (in PortProviderUtil) into a PrivilegedAction; there has to be an explanation (I assume that's on purpose, maybe someone from core team can help?)